backend/de.metas.business/src/main/sql/postgresql/ddl/functions/C_Invoice_Adv_Search_update.sql

```diff
@@ -0,0 +1,164 @@
+DROP FUNCTION IF EXISTS C_Invoice_Adv_Search_Update(
+    p_C_Invoice_IDs numeric[]
+)
+;
+
+
+DROP FUNCTION IF EXISTS C_Invoice_Adv_Search_Update()
+;
+CREATE FUNCTION C_Invoice_Adv_Search_Update(p_c_invoice_ids numeric[])
+    RETURNS TABLE
+            (
+                operation     character,
+                es_documentid character varying
+            )
+    LANGUAGE plpgsql
+AS
+$BODY$
+DECLARE
+    v_transactionId        text;
+    v_deleted_documentIds  varchar[];
+    v_inserted_documentIds varchar[];
+BEGIN
+    RAISE NOTICE 'p_C_Invoice_IDs: %', p_C_Invoice_IDs;
+    IF (p_C_Invoice_IDs IS NULL OR CARDINALITY(p_C_Invoice_IDs) <= 0) THEN
+        RAISE EXCEPTION 'At least one C_Invoice_ID shall be specified';
+    END IF;
+
+    v_transactionId = gen_random_uuid()::text;
+    RAISE NOTICE 'Transaction ID: %', v_transactionId;
```

Comment:
where is this `v_trasactionId` used?

Reply:
I thought it's a convention., since it was like this in the partner update function as well.
Dropped it.